src/olaf/identifier.rs

```diff
@@ -0,0 +1,176 @@
+//! The identifier of a participant in the Olaf protocol.
+
+use super::errors::DKGError;
+use core::cmp::Ordering;
+#[cfg(feature = "serde")]
+use core::fmt;
+use curve25519_dalek::Scalar;
+#[cfg(feature = "serde")]
+use serde::de::{self, Visitor};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+/// The identifier is represented by a Scalar.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Identifier(pub(crate) Scalar);
+
+#[cfg(feature = "serde")]
+impl Serialize for Identifier {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let scalar_bytes = self.0.to_bytes();
+        let scalar_hex = hex::encode(scalar_bytes);
+        serializer.serialize_str(&scalar_hex)
+    }
+}
+
+#[cfg(feature = "serde")]
+struct IdentifierVisitor;
```

Comment:
I'll remove serde from this repo eventually, so serde should not be used here, and should be avoided for crypto in general.
https://github.com/w3f/schnorrkel/issues/73

We do need serialization of course, but this should be done directly as binary, probably little endian.